pkgs/games/minecraft/default.nix

```diff
@@ -1,10 +1,27 @@
-{ stdenv, fetchurl, jre, libX11, libXext, libXcursor, libXrandr, libXxf86vm
+{ stdenv, fetchurl, makeDesktopItem
+, jre, libX11, libXext, libXcursor, libXrandr, libXxf86vm
 , mesa, openal
 , useAlsa ? false, alsaOss ? null }:
 
 assert useAlsa -> alsaOss != null;
 
-stdenv.mkDerivation {
+let
+  icon = fetchurl {
+    url = "https://hydra-media.cursecdn.com/minecraft.gamepedia.com/c/c5/Grass.png";
```

Comment:
@jagajaga  It comes from this page http://minecraft.gamepedia.com/File:Grass.png which says it is copyrighted by Mojang and has an attribution share alike license. I figure being associated with their own game can be considered attribution enough? I believe Arch Linux uses this same (or very similar) image, though they have it in the package archive itself.